modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -0,0 +1,131 @@
+
+    private static final String DEFAULT_TEST_TENANT_NAME = "test";
+
+    private static final String DEFAULT_USERNAME = "root";
+
+    private static final String DEFAULT_PASSWORD = "";
+
+    private static final String DEFAULT_DATABASE_NAME = "test";
+
+    private String tenantName = DEFAULT_TEST_TENANT_NAME;
+
+    private String driverClassName = "com.mysql.cj.jdbc.Driver";
+
+    public OceanBaseCEContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public OceanBaseCEContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPorts(SQL_PORT, RPC_PORT);
+    }
+
+    @Override
+    public String getDriverClassName() {
+        return driverClassName;
+    }
+
+    @Override
+    public String getJdbcUrl() {
+        return getJdbcUrl(DEFAULT_DATABASE_NAME);
+    }
+
+    public String getJdbcUrl(String databaseName) {
+        String additionalUrlParams = constructUrlParameters("?", "&");
+        String prefix = driverClassName.contains("mysql") ? "jdbc:mysql://" : "jdbc:oceanbase://";
+        return prefix + getHost() + ":" + getMappedPort(SQL_PORT) + "/" + databaseName + additionalUrlParams;
+    }
+
+    @Override
+    public String getDatabaseName() {
+        return DEFAULT_DATABASE_NAME;
+    }
+
+    @Override
+    public String getUsername() {
+        return DEFAULT_USERNAME + "@" + tenantName;
+    }
+
+    @Override
+    public String getPassword() {
+        return DEFAULT_PASSWORD;
+    }
+
+    @Override
+    protected String getTestQueryString() {
+        return "SELECT 1";
+    }
+
+    /**
+     * Set the non-system tenant to be created for testing.
+     *
+     * @param tenantName the name of tenant to be created
+     * @return this
+     */
+    public OceanBaseCEContainer withTenant(String tenantName) {
+        if (StringUtils.isEmpty(tenantName)) {
+            throw new IllegalArgumentException("Tenant name cannot be null or empty");
+        }
+        if (SYSTEM_TENANT_NAME.equals(tenantName)) {
+            throw new IllegalArgumentException("Tenant name cannot be " + SYSTEM_TENANT_NAME);
+        }
+        this.tenantName = tenantName;
+        return self();
+    }
+
+    /**
+     * Set the driver class name.
+     *
+     * @param driverClassName the driver class name
+     * @return this
+     */
+    public OceanBaseCEContainer withDriverClassName(String driverClassName) {
+        if (StringUtils.isEmpty(driverClassName)) {
+            throw new IllegalArgumentException("Driver class name cannot be null or empty");
+        }
+        if (!driverClassName.contains("mysql") && !driverClassName.contains("oceanbase")) {
+            throw new IllegalArgumentException("Driver class name should contains 'mysql' or 'oceanbase'");
+        }
+        this.driverClassName = driverClassName;
+        return self();
+    }
```

Comment:
I think something like [this](https://github.com/testcontainers/testcontainers-java/blob/f1b0b07b29594b65bd555e136ac4776b040a8294/modules/mysql/src/main/java/org/testcontainers/containers/MySQLContainer.java#L98-L106) can be done and then use it to detect the prefix. WDYT?

Reply:
Added `OceanBaseJdbcUtils` for it, please take a look.